src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -50,6 +50,7 @@ public function collect(Request $request, Response $response, \Exception $except
                 'enabled' => false,
                 'authenticated' => false,
                 'token_class' => null,
+                'provider_key' => null,
```

Comment:
@cordoval multiple firewalls can share the same authentication provider (using the ``context`` setting in the config). So the provider key may not be the firewall name.